india_stocks_api/internal/context.py

```
@@ -18,11 +19,100 @@
+        _CREDS_CACHE = {}
+
+    _CREDS_LOADED = True
+    return _CREDS_CACHE
+
+
+def _flush_creds() -> None:
+    """Persist in-memory credentials cache to creds.json."""
+    path = _get_creds_path()
+    try:
+        path.write_text(json.dumps(_CREDS_CACHE, indent=2), encoding="utf-8")
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)